specimen/wildlife-sounds-soundscapes/eml.xml

```diff
@@ -121,11 +118,11 @@ xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 http://rs.gbif.org/schema
                 </boundingCoordinates>
             </geographicCoverage>
             <taxonomicCoverage>
-                <generalTaxonomicCoverage>Animalia</generalTaxonomicCoverage>
+                <generalTaxonomicCoverage>Chiroptera (taxonomy based on the ASM Mammal Diversity Database).</generalTaxonomicCoverage>
```

Comment:
Idem

Reply:
er is inderdaad iets verwisseld tussen wildlife sounds en bats

Reply:
Heb dit aangepast en getest